examples/ai-slop-remover.claude.md

```
@@ -0,0 +1,135 @@
+_base: "{{ _base | default: 'main' }}"
+---
+
+# Remove AI Code Slop
+
+Identify and remove AI-generated code artifacts introduced in this branch.
+
+## Diff Analysis
+
+Analyzing changes against `{{ _base }}`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)